packages/plugin-core/src/commands/RandomNote.ts

```diff
@@ -57,10 +57,10 @@ export class RandomNoteCommand extends BasicCommand<
 
       return isMatch;
     };
+    const notesToPick = await engine.findNotesMeta({ excludeStub: true });
```

Comment:
can you add a TODO / warning: comment above this that this may be quite expensive if the user has a lot of notes?

I don't think this feature gets much usage so let's defer further changes.